src/rbac.rs

```diff
@@ -139,17 +140,26 @@ impl Authorization {
                         |p| p.matches(&ns),
                     );
 
-                    group_match &= m;
+                    if m {
+                        rule_match = true;
+                        break;
+                    }
                 }
 
-                if group.is_empty() {
-                    trace!(matches = true, "empty group");
+                if rule.is_empty() {
+                    rule_match = true;
+                    trace!(matches = rule_match, "empty rule");
                 } else {
-                    trace!(matches = group_match, "group");
+                    trace!(matches = rule_match, "rule");
+                }
+                group_match &= rule_match;
```

Comment:
Why it is  still `AND`

Reply:
For a "group" to match all elements (to/from/when in the authz policy) must match.

Reply:
Right, i misread it. I thought group_match means all groups match